src/Endpoint.js

```diff
@@ -220,63 +230,86 @@ const parseSchemaName = (ref) => {
 const Response = ({
   data, refs, name, response, first,
 }) => (
-    <Box border={first ? 'horizontal' : 'bottom'} pad={{ vertical: 'medium' }}>
-      <Box direction='row' pad={{ bottom: 'small' }} align='end'>
-        <Box basis='xxsmall'>
-          <Heading level={3} size='small' margin={{ vertical: 'small' }}>
-            <strong><code>{name}</code></strong>
-          </Heading>
-        </Box>
-        <Box flex={true} pad={{ horizontal: 'small' }} margin={{ vertical: 'small' }}>
-          <Markdown>
-            {sanitizeForMarkdown(response.description)}
-          </Markdown>
-        </Box>
+  <Box border={first ? 'horizontal' : 'bottom'} pad={{ vertical: 'medium' }}>
+    <Box direction='row' pad={{ bottom: 'small' }} align='end'>
+      <Box basis='xxsmall'>
+        <Heading level={3} size='small' margin={{ vertical: 'small' }}>
+          <strong><code>{name}</code></strong>
+        </Heading>
+      </Box>
+      <Box flex={true} pad={{ horizontal: 'small' }} margin={{ vertical: 'small' }}>
+        <Markdown>
+          {sanitizeForMarkdown(response.description)}
+        </Markdown>
       </Box>
-      {refs.schema && parseSchemaName(refs.schema.$ref) &&
-        <Box direction='column' align='start' pad={{ bottom: 'medium' }}>
-          <Text>
-            {'returns '}
-            <strong>
-              <RoutedAnchor
-                label={parseSchemaName(refs.schema.$ref)}
-                path={`/definition?name=${parseSchemaName(refs.schema.$ref)}`}
-              />
-            </strong>
-          </Text>
-        </Box>
-      }
-      {response.examples ?
-        Object.keys(response.examples).map(key =>
-          <Schema key={key} label={key} data={data} schema={response.examples[key]} />)
-        :
-        <Schema data={data} schema={getExample(response)} />
-      }
     </Box>
-  );
+    {refs.schema && parseSchemaName(refs.schema.$ref) &&
+      <Box direction='column' align='start' pad={{ bottom: 'medium' }}>
+        <Text>
+          {'returns '}
+          <strong>
+            <RoutedAnchor
+              label={parseSchemaName(refs.schema.$ref)}
+              path={`/definition?name=${parseSchemaName(refs.schema.$ref)}`}
+            />
+          </strong>
+        </Text>
+      </Box>
+    }
+    {response.examples ?
+      Object.keys(response.examples).map(key =>
+        <Schema key={key} label={key} data={data} schema={response.examples[key]} />)
+      :
+      <Schema data={data} schema={getExample(response)} />
+    }
+  </Box>
+);
+
+const getPath = (spec, data) => {
+  if (spec >= '3.0.1') {
```

Comment:
Wow, I can't believe this interprets properly... I suppose JS is typecasting the string to a number? This boggles my mind.

![Screen Shot 2019-10-08 at 8 54 04 AM](https://user-images.githubusercontent.com/5983843/66397190-4cdff000-e9a9-11e9-9134-0a5cf5788ee5.png)